lib/ret_web/controllers/api/v1/scene_controller.ex

```diff
@@ -0,0 +1,29 @@
+defmodule RetWeb.Api.V1.SceneController do
+  use RetWeb, :controller
+
+  alias Ret.Scene
+  alias Ret.Repo
+
+  # Limit to 1 TPS
+  plug(RetWeb.Plugs.RateLimit)
+
+  def show(conn, %{"id" => scene_sid}) do
```

Comment:
On second thoughts, not sure if this is the right pattern for API endpoints. Maybe SIDs should only be used in non-API URLs

Reply:
i think that sounds right